syntax="proto3";

service Backend {
    rpc load_image (img_path) returns (image);
}

##Â Mala definicion del metodo en el servicio del backend
service Backend {
    rpc run_model();
}

message image {
    string img_content = 1;
    int32 width = 2;
    int32 height = 3;
}

message img_path {
    string path = 1;
}

